<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>470</width>
    <height>426</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Empyrion Save</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="foldersGroup">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>451</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>Folders</string>
    </property>
    <widget class="QLabel" name="backupLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Backup Location</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="backupEdit">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>60</y>
       <width>251</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="backupBrowse">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>60</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="acceptDrops">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Browse</string>
     </property>
    </widget>
    <widget class="QLabel" name="saveLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Save Location</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="saveEdit">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>20</y>
       <width>251</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="saveBrowse">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>20</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="acceptDrops">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Browse</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="timerGroup">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>451</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string>Save Timer</string>
    </property>
    <widget class="QLabel" name="intervalLabel">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>00m00s</string>
     </property>
    </widget>
    <widget class="QSlider" name="timerSlider">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>20</y>
       <width>331</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <number>1800</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="controlGroup">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>451</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>Control Plane</string>
    </property>
    <widget class="QPushButton" name="saveButton">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>20</y>
       <width>81</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Save Settings</string>
     </property>
    </widget>
    <widget class="QPushButton" name="startButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="optionsGroup">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>451</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>Options</string>
    </property>
    <widget class="QCheckBox" name="saveZipCheck">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>111</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Save as Zip</string>
     </property>
    </widget>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>451</width>
      <height>71</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>470</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="statusTip">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
